code/index.html

```diff
@@ -26,14 +26,24 @@ <h1>Does the person have</h1>
         <optgroup label="hair color">
           <option value="brown">brown hair</option>
```

Comment:
`<option value="brown" selected>brown hair</option>` should fix your problem with the brown hairs 🤓